Makefile

```diff
@@ -18,7 +18,7 @@ yyyymmdd ?= $(today)
 buildroot ?= $(pwd)/buildroot/$(yyyymmdd)
 
 # The default chroot to build for.
-chroot ?= fedora-37-x86_64
+chroot ?= $(shell copr list-chroots | grep -P '^fedora-[0-9]+-x86_64' | sort | head -1)
```

Comment:
Maybe we should just use rawhide here?

Reply:
I don't think so. I mean, we're building the snapshots on rawhide as well, and they're about as stable as the rest.

Probably the main disadvantage of rawhide builds is that you likely have to download more updated packages than if you build for something like f37.